File: Python/Program/Petshop.py
class Petshop:
    __id = ""
    __namaProduk = ""
    __hargaProduk = 0
    __stokProduk = 0
    
    def __init__(self, id = "", namaProduk = "", hargaProduk = 0, stokProduk = 0):
        self.__id = id
        self.__namaProduk = namaProduk
        self.__hargaProduk = hargaProduk
        self.__stokProduk = stokProduk
        
    def getId(self):
        return self.__id

    def setId(self, id):
        self.__id = id 

    def getNamaProduk(self):
        return self.__namaProduk

    def setNamaProduk(self, namaProduk):
        self.__namaProduk = namaProduk
        
    def getHargaProduk(self):
        return self.__hargaProduk

    def setHargaProduk(self, hargaProduk):
        self.__hargaProduk = hargaProduk

    def getStokProduk(self):
        return self.__stokProduk

    def setStokProduk(self, stokProduk):
        self.__stokProduk = stokProduk

File: PHP/Program/Petshop.php
<?php

class Petshop {
    private $id = '';
    private $namaProduk = '';
    private $hargaProduk = 0;
    private $stokProduk = 0;
    private $fotoProduk = '';

    public function __construct($id = '', $namaProduk = '', $hargaProduk = 0, $stokProduk = 0, $fotoProduk = '')
    {
        $this->id = $id;
        $this->namaProduk = $namaProduk;
        $this->hargaProduk = $hargaProduk;
        $this->stokProduk = $stokProduk;
        $this->fotoProduk = $fotoProduk;
    }

    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getNamaProduk() {
        return $this->namaProduk;
    }

    public function setNamaProduk($namaProduk) {
        $this->namaProduk = $namaProduk;
    }

    public function getHargaProduk() {
        return $this->hargaProduk;
    }

    public function setHargaProduk($hargaProduk) {
        $this->hargaProduk = $hargaProduk;
    }

    public function getStokProduk() {
        return $this->stokProduk;
    }

    public function setStokProduk($stokProduk) {
        $this->stokProduk = $stokProduk;
    }

    public function getFotoProduk() {
        return $this->fotoProduk;
    }

    public function setFotoProduk($fotoProduk) {
        $this->fotoProduk = $fotoProduk;
    }

    public function __destruct(){}
}
?>